src/main/java/com/library/model/entity/mongo/Bookshelf.java

```diff
@@ -0,0 +1,13 @@
+package com.library.model.entity.mongo;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class Bookshelf {
```

Comment:
CamelCase is needed.

And why we need this been?

Reply:
To know where the book was before getting into the storage